pytorch_lightning/accelerators/cpu_accelerator.py

```diff
@@ -53,10 +53,8 @@ def setup(self, model):
         self.trainer.model = model
 
     def train(self):
-        model = self.trainer.model
-
-        # set up training routine
-        self.trainer.train_loop.setup_training(model)
+        # set up trainer
```

Comment:
this comment adds no value. same for the other accelerators

```suggestion
```

Reply:
yeah.. will update